checkov/common/checks/object_registry.py

```diff
@@ -0,0 +1,179 @@
+from checkov.common.checks.base_check_registry import BaseCheckRegistry
+from checkov.common.models.enums import CheckResult
+from checkov.yaml_doc.enums import BlockType
+
+
+class Registry(BaseCheckRegistry):
```

Comment:
can that be resolved using inheritance/ usage of the yaml or json registry?

Reply:
why not creating a new common enums for both json and yaml?